conf/base.config

```diff
@@ -83,12 +83,23 @@ process {
 
     // PYTHON BASED MODULES WHICH SHOULDN'T NEED MORE THAN 1 CORE AND A MIDDLING AMOUNT OF MEMORY
     // WILL BE REVIEWED
-    withName: 'VALIDATE_TAXID|TRAILINGNS|GC_CONTENT|GET_KMERS_PROFILE|PARSE_FCSGX_RESULT|ASCC_MERGE_TABLES|GET_LARGEST_SCAFF|KMER_COUNT_DIM_REDUCTION|KMER_COUNT_DIM_REDUCTION_COMBINE_CSV|REFORMAT_DIAMOND_OUTFMT6|CONVERT_TO_HITS_FILE|DIAMOND_BLAST_CHUNK_TO_FULL|GENERATE_SAMPLESHEET' {
+    withName: 'VALIDATE_TAXID|TRAILINGNS|GC_CONTENT|PARSE_FCSGX_RESULT|ASCC_MERGE_TABLES|GET_LARGEST_SCAFF|KMER_COUNT_DIM_REDUCTION|KMER_COUNT_DIM_REDUCTION_COMBINE_CSV|REFORMAT_DIAMOND_OUTFMT6|CONVERT_TO_HITS_FILE|DIAMOND_BLAST_CHUNK_TO_FULL|GENERATE_SAMPLESHEET' {
         cpus    = { 1                           }
         memory  = { 10.GB    * task.attempt     }
         time    = { 5.h      * task.attempt     }
     }
 
+    withName: "GET_KMERS_PROFILE" {
+        //200mb genome becomes a 2.2 modifier
+        // 1.2 * 15Gb * task.attempt
+        // 1Gb genome becomes a 3x modifier
+        // etc
+        // FOR TESTING!!!
+        cpus    = { 1                           }
+        memory  = { ((fasta.size / 1e9) + 2.GB * 15.GB) * task.attempt }
```

Comment:
I think the memory usage has less to do with the size and more to do with the number of sequences - for each sequence, you need to hold 4^7 numbers in memory (plus more for the processing). For a fasta with 100k contigs, assuming the default 64 bit float in pandas:

`(4^7) kmers * 1e5 contigs * 64 bytes / 1e9` = 104 Gb RAM required.